polish CHANGES and add note about 0.6.1 (#138)
pangeo-data/xESMF: CHANGES.rst

What's new
==========

0.6.2 (23-11-2021)
------------------

Bug fixes
~~~~~~~~~

> - The introduction of `sparse`, with `numba` under the hood, restricted input data to little-endian dtypes. For big-endian dtypes, xESMF switches back to using scipy (:pull:`125`). By `Pascal Bourgault <https://github.com/aulemahal>`_

Reviewer: ```suggestion
- The introduction of `sparse`, with `numba` under the hood, restricted input data to little-endian dtypes. For big-endian dtypes, xESMF will convert to little-endian, regrid and convert back (:pull:`135`). By `Pascal Bourgault <https://github.com/aulemahal>`_
```
